### PR TITLE
Twios 28

### PR DESCRIPTION
**Pull request by emmasofiah2o**
Add 28th twios to website.

### CODE REVIEW

File: src/twios/2023-08-22.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+## typesafe-i18n
+
+- [ivanhofer/typesafe-i18n-demo-sveltekit]
+  [#40](https://github.com/ivanhofer/typesafe-i18n-demo-sveltekit/pull/40) docs:
+  fix Warning box in README.md ([@oscard0m])
+
+## Unknown
+
+- [getzola/zola] [#2289](https://github.com/getzola/zola/pull/2289) docs: update
+  links to new Tera docs URL ([@oscard0m])
```
Comment (oscard0m) [inline src/twios/2023-08-22.md:79]: category: zola

File: src/twios/2023-08-22.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+- [mansona/ember-inline-svg]
+  [#1](https://github.com/mansona/ember-inline-svg/pull/1) update
+  @embroider/test-setup ([@mansona])
+- [minutebase/ember-inline-svg]
+  [#128](https://github.com/minutebase/ember-inline-svg/pull/128) update
+  @embroider/test-setup ([@mansona])
+- [natemoo-re/astro-capo] [#2](https://github.com/natemoo-re/astro-capo/pull/2)
+  docs: add link to capo.js in README.md ([@oscard0m])
```
Comment (oscard0m) [inline src/twios/2023-08-22.md]: category: Astro

**Review by oscard0m — APPROVED** [review]

File: src/twios/2023-08-22.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+- [ivanhofer/typesafe-i18n-demo-sveltekit]
+  [#40](https://github.com/ivanhofer/typesafe-i18n-demo-sveltekit/pull/40) docs:
+  fix Warning box in README.md ([@oscard0m])
+
+## Unknown
+
+- [getzola/zola] [#2289](https://github.com/getzola/zola/pull/2289) docs: update
+  links to new Tera docs URL ([@oscard0m])
+- [mansona/ember-inline-svg]
```
Comment (mansona) [inline src/twios/2023-08-22.md]: ignore this one since it's a fork

File: src/twios/2023-08-22.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+  [#1](https://github.com/mansona/ember-inline-svg/pull/1) update
```
Comment (mansona) [inline src/twios/2023-08-22.md]: add this one to Ember.js

**Pull request merged**